dags.yaml

```
@@ -1394,3 +1394,23 @@ bqetl_mobile_feature_usage:
   schedule_interval: 0 6 * * *
   tags:
     - impact/tier_3
+
+bqetl_telemetry_dev_cycle:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - leli@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
```

```suggestion
    email_on_retry: false
```
I assume we only want a notification on a failure?